packages/bot-web-ui/src/components/dashboard/joyride-config.tsx

```diff
@@ -5,20 +5,35 @@ import { Text } from '@deriv/components';
 type TJoyrideConfig = Record<'showProgress' | 'showSkipButton' | 'spotlightClicks' | 'disableBeacon', boolean>;
 
 type TStep = {
-    label: string | boolean;
-    content: string;
+    label?: string;
+    content: TContent[];
 };
 
-const Step = ({ label, content }: TStep) => (
-    <div className='db-tour'>
-        <Text line_height='s' color='prominent' as='h2' weight='bold'>
-            {label}
-        </Text>
-        <Text line_height='s' color='prominent' as='p'>
-            {content}
-        </Text>
-    </div>
-);
+type TContent = {
+    contentData: string;
+};
+
+export const Step = ({ label, content }: TStep) => {
+    return (
+        <div className='db-tour' style={{}}>
+            <Text line_height='xl' as='p' weight='bold' color='colored-background'>
+                {label}
+            </Text>
+            {content.map(data => {
+                const { contentData } = data;
```

Comment:
can we make it `content_data`??? and do we really need it?

Reply:
removed contentData.